ee/backend/app/src/main/kotlin/io/tolgee/ee/service/prompt/PromptDefaultService.kt

```diff
@@ -0,0 +1,43 @@
+package io.tolgee.ee.service.prompt
+
+import io.tolgee.dtos.request.prompt.PromptDto
+import io.tolgee.service.LlmPropertiesService
+import org.springframework.stereotype.Service
+
+@Service
+class PromptDefaultService(
```

Comment:
PromptDefault service sound like it's Some kind of default service prompts, not service which handles the default prompt.

Not sure if this way of naming is always right. 🤷 Are there some documented conventions or some industry standard in this?

Reply:
I'm not sure either, it's like a helper class, so it might be called helper.

Reply:
Make it helpers, without @Service annotation